deploy/deploy_test.go

```diff
@@ -484,11 +484,11 @@ func TestMetalLBSpec(t *testing.T) {
 			IPCount: 20,
 		},
 		execer: exec.NewFakeExecer(nil, nil, nil),
-		wantCM: `address-pools:
-    - name: default
-      protocol: layer2
-      addresses:
-        - 172.18.0.50 - 172.18.0.70
+		wantCM: `AddressPools:
```

Comment:
with this change we should delete https://github.com/openconfig/kne/blob/main/examples/metallb/config_map.yaml (since its no longer very useful and now OOD)

Reply:
will do in next pr didn't even know that was still a thing